src/ucp/core/ucp_am.c

```diff
@@ -949,6 +949,27 @@ static UCS_F_ALWAYS_INLINE uint16_t ucp_am_send_nbx_get_op_flag(uint32_t flags)
     return 0;
 }
 
+static UCS_F_ALWAYS_INLINE ucs_status_t
+ucp_am_params_check_memh(const ucp_request_param_t *param, uint32_t *flags_p)
+{
+    if (ucs_unlikely(param->op_attr_mask & UCP_OP_ATTR_FIELD_MEMH)) {
+        /* Suppress Coverity warning that memh could be NULL and dereferenced
+         */
+        ucs_assert(param->memh != NULL);
+
+        if (ucs_unlikely(param->memh->flags & UCP_MEMH_FLAG_IMPORTED)) {
+            if (ucs_unlikely(ENABLE_PARAMS_CHECK &&
```

Comment:
ENABLE_PARAMS_CHECK can be outside of ucs_unlikely

Reply:
done